bin/__tests__/getIntegrationAccessToken.test.js

```diff
@@ -13,7 +13,9 @@
 const proxyquire = require('proxyquire');
 
 const METASCOPES = [
+  'ent_reactor_sdk',
   'ent_reactor_extension_developer_sdk',
+  'ent_reactor_admin',
```

Comment:
@spencerroan is saying that `ent_reactor_admin` is not new, so it shouldn't be added. I don't really understand the metascopes enough to say otherwise.